types/aws-lambda/trigger/cloudwatch-events.d.ts

```diff
@@ -1,19 +1,10 @@
-import { Handler } from "../handler";
+import { EventBridgeHandler, EventBridgeEvent } from '../trigger/eventbridge';
 
-export type ScheduledHandler = Handler<ScheduledEvent, void>;
+export type ScheduledHandler = EventBridgeHandler<string, ScheduledEvent, void>;
 
 // TODO: generic cloudwatch event types.
 
 /**
  * https://docs.aws.amazon.com/lambda/latest/dg/with-scheduled-events.html
  */
-export interface ScheduledEvent {
-    account: string;
-    region: string;
-    detail: any;
-    'detail-type': string;
-    source: string;
-    time: string;
-    id: string;
-    resources: string[];
-}
+export interface ScheduledEvent extends EventBridgeEvent<string, any> {}
```

Comment:
Hmm, I'm torn. Really TDetailType here should be `"Scheduled Event"`, since that's [what you get](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html#schedule_event_type) if we're using `detail-type` to distinguish different events to the same lambda, but that is a (small, unlikely) breaking change, and is pretty unlikely to be useful. I'd lean on leaving it like this, it's easy enough with this PR to do it yourself.

Reply:
Ok I changed it, I also added an example in the event bridge test to a multiple event handler. I didn't manage to use the EventBridgeHandler as I am not sure how I could manage the nested generic type in TS .
I would need something like

`
export type EventBusHandler<TEvent extends EventBridgeEvent, TResult> = BaseHandler<TEvent, TResult> 
`

but AFAIK it is not possible, or at least I don't know how to do it ;-)